src/minter-suite-mapping.ts

```diff
@@ -692,54 +680,30 @@ export function handleUnregisteredNFTAddressV2(
 // Because AssemblyScript does not support union types, we need to manually type check inside each method, to ensure correct usage.
 // For any questions reach out to @jon or @ryley-o.eth. or see the following document https://docs.google.com/document/d/1XSxl04eJyTxc_rbj6cmq-j00zaYDzApBBLT67JXtaOw/edit?disco=AAAAZa8xp-Q
```

Comment:
nit: can we update this function's documentation a bit? would be great to define what type of values are allowed for T, and what types of values are allowed for C.

Reply:
For sure! I listed out the T values but it looks like we already called out the supported C values. But actually I noticed the documentation showed ProjectMinterConfiguration as a valid type -- since we're type-checking for Minter & Project [I changed this to "Project"](https://github.com/ArtBlocks/artblocks-subgraph/pull/135/commits/ccfdbd4ea1fdfd97e2a2c0b32961f15658670f28#diff-a1077c7a5a21bf6e3a06b3124ce379bd5374fef2987d7164b250ae14fad15c56R680) but I wanted to flag this in case we're missing one maybe?
@ryley-o

Reply:
Nvm I [changed this back](https://github.com/ArtBlocks/artblocks-subgraph/pull/135/commits/18a42c918ab3e9dc980f3a162fbe1f06919b4367) since it looks like we're only using this handler on Minters & ProjectMinterConfigs